nvflare/apis/impl/controller.py

```diff
@@ -65,14 +64,21 @@ def _check_inputs(task: Task, fl_ctx: FLContext, targets: Union[List[Client], Li
                 )
 
 
+def _get_client_task(target, task: Task):
```

Comment:
this method `_get_client_task` could either be move out as a function or add `@classmethod` decorator to it.
Since it does not use any instance variable.

I found move out more favorable because this method could potentially be used in other scenarios.